packages/editor/src/components/sidebar/constants.js

```diff
@@ -0,0 +1,4 @@
+export const sidebars = {
+	document: 'edit-post/document',
+	block: 'edit-post/block',
```

Comment:
Are we keeping the old namespace for now? Let's note that we should update these when sidebar-related store actions are also unified.

Reply:
We can't change these for BC reasons unfortunately. We discussed this with @fabiankaegy in a previous PR and this was the best tradeoff